<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" total="4" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="223802" started-at="2018-08-03T09:31:41Z" finished-at="2018-08-03T09:35:25Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="223802" started-at="2018-08-03T09:31:41Z" finished-at="2018-08-03T09:35:25Z">
      <class name="com.Tests.CallTest">
        <test-method status="PASS" signature="startUp()[pri:0, instance:com.Tests.CallTest@3532ec19]" name="startUp" is-config="true" duration-ms="75848" started-at="2018-08-03T15:01:42Z" finished-at="2018-08-03T15:02:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startUp -->
        <test-method status="FAIL" signature="newCalladding(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.Tests.CallTest@3532ec19]" name="newCalladding" duration-ms="21703" started-at="2018-08-03T15:02:58Z" data-provider="AddCall" finished-at="2018-08-03T15:03:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Y]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[03-Aug-2018 00:18]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[15 Minutes]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Via Email]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Call 1 Added]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
	The method CaptureScreenshot(WebDriver, String, String) in the type ScreenShot is not applicable for the arguments (WebDriver)
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
	The method CaptureScreenshot(WebDriver, String, String) in the type ScreenShot is not applicable for the arguments (WebDriver)

	at com.Utilities.WebEventListener.onException(WebEventListener.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver$1.invoke(EventFiringWebDriver.java:86)
	at com.sun.proxy.$Proxy7.onException(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver$EventFiringWebElement$1.invoke(EventFiringWebDriver.java:378)
	at com.sun.proxy.$Proxy9.click(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver$EventFiringWebElement.click(EventFiringWebDriver.java:389)
	at com.BasePage.Basepage.Click(Basepage.java:187)
	at com.Tests.LoginPageTest.Login(LoginPageTest.java:74)
	at com.Tests.CallTest.newCalladding(CallTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- newCalladding -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:com.Tests.CallTest@3532ec19]" name="TearDown" is-config="true" duration-ms="1120" started-at="2018-08-03T15:03:20Z" finished-at="2018-08-03T15:03:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method status="PASS" signature="startUp()[pri:0, instance:com.Tests.CallTest@3532ec19]" name="startUp" is-config="true" duration-ms="14116" started-at="2018-08-03T15:03:21Z" finished-at="2018-08-03T15:03:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startUp -->
        <test-method status="FAIL" signature="newCalladding(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.Tests.CallTest@3532ec19]" name="newCalladding" duration-ms="20853" started-at="2018-08-03T15:03:35Z" data-provider="AddCall" finished-at="2018-08-03T15:03:56Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Y]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[04-Aug-2018 00:18]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[30 Minutes]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Via Text Message]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Call 2 Added]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
	The method CaptureScreenshot(WebDriver, String, String) in the type ScreenShot is not applicable for the arguments (WebDriver)
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
	The method CaptureScreenshot(WebDriver, String, String) in the type ScreenShot is not applicable for the arguments (WebDriver)

	at com.Utilities.WebEventListener.onException(WebEventListener.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver$1.invoke(EventFiringWebDriver.java:86)
	at com.sun.proxy.$Proxy7.onException(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver$EventFiringWebElement$1.invoke(EventFiringWebDriver.java:378)
	at com.sun.proxy.$Proxy9.click(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver$EventFiringWebElement.click(EventFiringWebDriver.java:389)
	at com.BasePage.Basepage.Click(Basepage.java:187)
	at com.Tests.LoginPageTest.Login(LoginPageTest.java:74)
	at com.Tests.CallTest.newCalladding(CallTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- newCalladding -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:com.Tests.CallTest@3532ec19]" name="TearDown" is-config="true" duration-ms="857" started-at="2018-08-03T15:03:56Z" finished-at="2018-08-03T15:03:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method status="PASS" signature="startUp()[pri:0, instance:com.Tests.CallTest@3532ec19]" name="startUp" is-config="true" duration-ms="35850" started-at="2018-08-03T15:03:56Z" finished-at="2018-08-03T15:04:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startUp -->
        <test-method status="FAIL" signature="newCalladding(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.Tests.CallTest@3532ec19]" name="newCalladding" duration-ms="20114" started-at="2018-08-03T15:04:32Z" data-provider="AddCall" finished-at="2018-08-03T15:04:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[N]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[05-Aug-2018 00:18]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[45 Minutes]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Via Email]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Call 3 Added]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
	The method CaptureScreenshot(WebDriver, String, String) in the type ScreenShot is not applicable for the arguments (WebDriver)
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
	The method CaptureScreenshot(WebDriver, String, String) in the type ScreenShot is not applicable for the arguments (WebDriver)

	at com.Utilities.WebEventListener.onException(WebEventListener.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver$1.invoke(EventFiringWebDriver.java:86)
	at com.sun.proxy.$Proxy7.onException(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver$2.invoke(EventFiringWebDriver.java:111)
	at com.sun.proxy.$Proxy8.perform(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver.perform(EventFiringWebDriver.java:342)
	at org.openqa.selenium.interactions.Actions$BuiltAction.perform(Actions.java:638)
	at com.BasePage.Basepage.Mousehover(Basepage.java:592)
	at com.Tests.CallTest.newCalladding(CallTest.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- newCalladding -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:com.Tests.CallTest@3532ec19]" name="TearDown" is-config="true" duration-ms="730" started-at="2018-08-03T15:04:52Z" finished-at="2018-08-03T15:04:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method status="PASS" signature="startUp()[pri:0, instance:com.Tests.CallTest@3532ec19]" name="startUp" is-config="true" duration-ms="17448" started-at="2018-08-03T15:04:53Z" finished-at="2018-08-03T15:05:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startUp -->
        <test-method status="FAIL" signature="newCalladding(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.Tests.CallTest@3532ec19]" name="newCalladding" duration-ms="12502" started-at="2018-08-03T15:05:11Z" data-provider="AddCall" finished-at="2018-08-03T15:05:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[N]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[06-Aug-2018 00:18]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[1 Hour]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Via Text Message]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Call 4 Added]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
	The method CaptureScreenshot(WebDriver, String, String) in the type ScreenShot is not applicable for the arguments (WebDriver)
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
	The method CaptureScreenshot(WebDriver, String, String) in the type ScreenShot is not applicable for the arguments (WebDriver)

	at com.Utilities.WebEventListener.onException(WebEventListener.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver$1.invoke(EventFiringWebDriver.java:86)
	at com.sun.proxy.$Proxy7.onException(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver$2.invoke(EventFiringWebDriver.java:111)
	at com.sun.proxy.$Proxy8.perform(Unknown Source)
	at org.openqa.selenium.support.events.EventFiringWebDriver.perform(EventFiringWebDriver.java:342)
	at org.openqa.selenium.interactions.Actions$BuiltAction.perform(Actions.java:638)
	at com.BasePage.Basepage.Mousehover(Basepage.java:592)
	at com.Tests.CallTest.newCalladding(CallTest.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- newCalladding -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:com.Tests.CallTest@3532ec19]" name="TearDown" is-config="true" duration-ms="1468" started-at="2018-08-03T15:05:23Z" finished-at="2018-08-03T15:05:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- com.Tests.CallTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
